packages/grafana-ui/src/components/DataLinks/DataLinkSuggestions.tsx

```diff
@@ -87,6 +87,7 @@ export const DataLinkSuggestions = ({ suggestions, ...otherProps }: DataLinkSugg
           <DataLinkSuggestionsList
             {...otherProps}
             suggestions={groupedSuggestions[key]}
+            // eslint-disable-next-line @grafana/no-untranslated-strings
             label={`${capitalize(key)}`}
```

Comment:
if you just make this label={capitalize(key)} does it resolve the lint error?

Reply:
it does! 🚀